lib/ssl/src/ssl_cipher.erl

```diff
@@ -560,10 +560,20 @@ hash_size(sha384) ->
 hash_size(sha512) ->
     64.
 
-is_supported_sign({Hash, rsa} = SignAlgo, HashSigns) ->
+%% Handle RSA and RSA_PSS_RSAE
```

Comment:
should we mark this change as potentially incompatible?

Reply:
No it is a bug fix. Bug caused by sub-optimizations made in a previous refactor.

Reply:
The internal support issue is indicated in the branch name